sirepo/package_data/template/cloudmc/parameters.py.jinja

```diff
@@ -1,3 +1,6 @@
+{% if incomplete_data_msg != "" %}
+# Unable to generate sim due to: {{ incomplete_data_msg }}
```

Comment:
I think just testing for `if incomplete_data_msg` is fine. A non-empty string in python is True.

What about raising an error? If someone tries to run the file w/o looking at it if it just has the comment it will look like it succeeded but didn't do anything. Raising an error I think will help them more quickly see what's wrong.

Reply:
@moellep what do you think about generating an error that will be caught by the parser in the parameters.py? That way _generate_parameters() will run and the export will work, but if the user tries to run the sim it will be clear what is wrong

Reply:
@moellep ping

Reply:
Yes, I think that makes sense, something like this:
```python
{% if incomplete_data_msg != "" %}
raise AssertionError('{{ incomplete_data_msg }}')
{% else %}
```

Reply:
Also, along these lines, I don't think you need to include incomplete_data_msg in the jinja, just return the string directly:
```python
    if v.incomplete_data_msg:
        return v.incomplete_data_msg
    return template_common.render_jinja(
        SIM_TYPE,
        v,
    )
```